Gopkg.lock

```diff
@@ -1403,10 +1403,8 @@
     "github.com/yanniszark/go-nodetool/nodetool",
     "k8s.io/api/apps/v1",
     "k8s.io/api/apps/v1beta1",
```

Comment:
I'm wondering why this dependency is still here after switching to apps/v1.
Any idea why it is still a dependency? Is it used by "old" or somewhere in test code?